source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py

```diff
@@ -988,7 +988,12 @@ def transform_points(
 
 @torch.jit.script
 def unproject_depth(depth: torch.Tensor, intrinsics: torch.Tensor) -> torch.Tensor:
-    r"""Unproject depth image into a pointcloud.
+    r"""Unproject depth image into a pointcloud. This method assumes that depth
```

Comment:
Please follow google doc-style. The first line here is a one-line sumary. Everything else moves to a new para.